src/objects/zcl_abapgit_object_shi3.clas.abap

```diff
@@ -323,10 +323,10 @@ CLASS zcl_abapgit_object_shi3 IMPLEMENTATION.
         jump_se43( ).
```

Comment:
`rv_exit` should go into the two `when` branches.

Reply:
good catch. Fixed.